features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/Base/RouteMenuMapping.jsx

```diff
@@ -34,16 +34,16 @@ import GatewayEnvironments from 'AppComponents/GatewayEnvironments/ListGWEnviorn
 import AdvancedThrottlePolicies from 'AppComponents/Throttling/Advanced';
 import CustomThrottlingPolicies from 'AppComponents/Throttling/Custom';
 import TenantTheme from 'AppComponents/TenantTheme/UploadTheme';
-import ListDetectedBotData from 'AppComponents/BotDetection/DetectedBotData/ListDetectedBotData';
-import ListEmails from 'AppComponents/BotDetection/EmailConfig/ListEmails';
+// import ListDetectedBotData from 'AppComponents/BotDetection/DetectedBotData/ListDetectedBotData';
```

Comment:
any reason for commenting this rather than removing the code?

Reply:
We will have to enable these and support thr functionality once the Alerts and Bot detection feature is supported in Choreo Analytics.

Reply:
yes, let's remove it and add it back when needed?

Reply:
+1

Reply:
Fixed with 023f06bbccadc483d4b82077ae43a6b44a6ccecd